neuraxle/hyperparams/distributions.py

```diff
@@ -309,7 +309,7 @@ def rvs(self):
 
         :return: True or False (random).
         """
-        return np.random.choice([False, True], p=[1 - self.proba_is_true, self.proba_is_true])
+        return bool(np.random.choice([False, True], p=[1 - self.proba_is_true, self.proba_is_true]))
```

Comment:
I would have thought it'd already work.

Reply:
ah, the return type was a numpy array of 1 instead of a boolean! I see. Are you sure the cast is good, don't you need to do [0] or something before bool() ? If not then this is resolved.

Reply:
yes the hyperparams json repository would not have worked otherwise @guillaume-chevalier